src/Arriba/Arriba/Diagnostics/Observability/ArribaObservationContext.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Arriba.Diagnostics.Observability
+{
+    public class ArribaObservationContext : IObservationContext
+    {
+        Guid _contextId;
+        Guid _sequenceId;
+
+        public Guid ContextId => _contextId;
```

Comment:
don't do this unless you need a volatile member. Instead use a {get;} only property that can only be initialized in the constructor.
```suggestion
        public Guid ContextId {get;}
```

Reply:
Fixed, thanks